tests/integration/test_decompositions_integration.py

```diff
@@ -575,7 +575,7 @@ def test_CXgate_decomp_equal(self, setup_eng, s, tol):
         eng.run(prog)
         assert np.all(eng.backend.is_vacuum(tol))
 
-    @pytest.mark.parametrize('s', np.linspace(0., 0.6, 5))
+    @pytest.mark.parametrize('s', np.linspace(-0.5, 0.6, 5))  # FIXME include 0 in testset
```

Comment:
same question

Reply:
Same answer :)

Reply:
Also here (adding this information to the FIXME comment)

Reply:
Done.